source/_patterns/02-molecules/card/_card.scss

```diff
@@ -16,20 +18,51 @@ $_config_schemes: (
     action-button: (
       "jcc-button--secondary",
     ),
+    action-button-justify: "justify",
     icon-color: primary-light,
+    overlay-bgcolors: (
```

Comment:
I wanted to just leave them there since they aren't using it anyway and we can change them later but at least there are placeholders.  What do you think?

Reply:
Regardless, you need placeholders. I think the designers won't want the Pro colors available to other themes but a TBD for other theme regardless. I don't know if null or "transparent" will work either. Sometimes they do. Your call.

Reply:
Yeah, I'm not sure what is best.  I could make all of the local/base colors to be the same.  Since the color name has to be passed to the twig, it probably makes sense to keep them in the theme section to avoid confusion?